generator/generator.go

```diff
@@ -394,6 +414,16 @@ func (g *Generator) parseEnum(ts *ast.TypeSpec) (*Enum, error) {
 	return enum, nil
 }
 
+func increment(d interface{}) interface{} {
+	switch d.(type) {
+	case uint64:
+		return d.(uint64) + 1
+	case int64:
+		return d.(int64) + 1
+	}
```

Comment:
Just noting that type assertion switch cases can do the conversion for you:

```suggestion
switch v := d.(type) {
case uint64:
  return v + 1
case int64:
  return v + 1
}
```

Reply:
Heh, well, I do like the elegant solution, making sure the signedness of the underlying type get used internally so any hardcoded values will have to match.

Reply:
Yeah, had about 3 or 4 iterations before I landed on this one. Definitely felt like the right level of safety and flexibility. Thanks for double checking my work!